### PR TITLE
[MRG] Adding updated documentation for Nihon Reader

### PR DESCRIPTION
**Pull request by adam2392**
#### Reference issue
Closes: #8319 

#### What does this implement/fix?
Adds documentation to document NK data files.

### CODE REVIEW

File: tutorials/io/plot_20_reading_eeg_data.py [hunk 1]
```diff
@@ -183,6 +183,27 @@
...
+Nihon Kohden EEG data (.EEG, .21E, .PNT, .LOG)
+======================================================================
+
+EEG data from the Nihon Kohden (NK) system can be read in using the
+:func:`mne.io.read_raw_nihon` function.
+
+The following file extensions are read in.
+- The ``.EEG`` file contains the actual raw EEG data.
+- The ``.PNT`` file contains the metadata related to the recording, such
+as the measurement date.
```
Comment (larsoner) [inline tutorials/io/plot_20_reading_eeg_data.py:195]:
```suggestion
  as the measurement date.
```
RST is picky, see how it rendered:

https://22536-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/io/plot_20_reading_eeg_data.html#nihon-kohden-eeg-data-eeg-21e-pnt-log

File: tutorials/io/plot_20_reading_eeg_data.py [hunk 2]
```diff
@@ -183,6 +183,27 @@
 sometimes from version to version. Please submit an issue, or
 pull request if you encounter a problem.
 
+Nihon Kohden EEG data (.EEG, .21E, .PNT, .LOG)
+======================================================================
+
+EEG data from the Nihon Kohden (NK) system can be read in using the
+:func:`mne.io.read_raw_nihon` function.
+
+The following file extensions are read in.
```
Comment (larsoner) [inline tutorials/io/plot_20_reading_eeg_data.py:192]:
```suggestion
The following file extensions are read in.

```

File: tutorials/io/plot_20_reading_eeg_data.py [hunk 3]
```diff
@@ -183,6 +183,27 @@
...
+- The ``.EEG`` file contains the actual raw EEG data.
+- The ``.PNT`` file contains the metadata related to the recording, such
+as the measurement date.
+- The ``.LOG`` file contains annotations for the recording.
+- The ``.21E`` file contains the channel and electrode
+recording system information.
```
Comment (larsoner) [inline tutorials/io/plot_20_reading_eeg_data.py:198]:
```suggestion
  recording system information.
```

**Review by larsoner — COMMENTED** [review]
Can you add a note about this doc update to `latest.inc`?

File: tutorials/io/plot_20_reading_eeg_data.py [hunk 4]
```diff
@@ -183,6 +183,28 @@
 sometimes from version to version. Please submit an issue, or
 pull request if you encounter a problem.
 
+Nihon Kohden EEG data (.EEG, .21E, .PNT, .LOG)
+======================================================================
```
Comment (hoechenberger) [inline tutorials/io/plot_20_reading_eeg_data.py:187]:
```suggestion
Nihon Kohden EEG data (.EEG, .21E, .PNT, .LOG)
==============================================
```

File: mne/io/nihon/nihon.py [hunk 5]
```diff
@@ -29,7 +29,7 @@ def read_raw_nihon(fname, preload=False, verbose=None):
     Parameters
     ----------
     fname : str
-        Path to the Nihon Kohden data file (.eeg).
+        Path to the Nihon Kohden data file (.EEG).
```
Comment (hoechenberger) [inline mne/io/nihon/nihon.py:32]:
```suggestion
        Path to the Nihon Kohden data file (``.EEG``).
```

File: tutorials/io/plot_20_reading_eeg_data.py [hunk 6]
```diff
@@ -183,6 +183,28 @@
...
+The following file extensions are read in:
+
+- The ``.EEG`` file contains the actual raw EEG data.
+- The ``.PNT`` file contains the metadata related to the recording, such
+  as the measurement date.
+- The ``.LOG`` file contains annotations for the recording.
+- The ``.21E`` file contains the channel and electrode
+  recording system information.
+
+The .11D, .CMT, .CN2, .EDF files are not currently read in.
```
Comment (hoechenberger) [inline tutorials/io/plot_20_reading_eeg_data.py:201]:
```suggestion
Reading ``.11D``, ``.CMT``, ``.CN2``, and ``.EDF`` files is currently not
supported.
```

File: tutorials/io/plot_20_reading_eeg_data.py [hunk 7]
```diff
@@ -183,6 +183,28 @@
 sometimes from version to version. Please submit an issue, or
 pull request if you encounter a problem.
 
+Nihon Kohden EEG data (.EEG, .21E, .PNT, .LOG)
+======================================================================
+
+EEG data from the Nihon Kohden (NK) system can be read in using the
```
Comment (hoechenberger) [inline tutorials/io/plot_20_reading_eeg_data.py:189]:
```suggestion
EEG data from the Nihon Kohden (NK) system can be read using the
```

File: tutorials/io/plot_20_reading_eeg_data.py [hunk 8]
```diff
@@ -183,6 +183,28 @@
...
+:func:`mne.io.read_raw_nihon` function.
+
+The following file extensions are read in:
```
Comment (hoechenberger) [inline tutorials/io/plot_20_reading_eeg_data.py:192]:
```suggestion
Files with the following extensions are supported:
```

File: tutorials/io/plot_20_reading_eeg_data.py [hunk 9]
```diff
@@ -183,6 +183,28 @@
...
+- The ``.PNT`` file contains the metadata related to the recording, such
+  as the measurement date.
+- The ``.LOG`` file contains annotations for the recording.
+- The ``.21E`` file contains the channel and electrode
+  recording system information.
+
+The .11D, .CMT, .CN2, .EDF files are not currently read in.
+
+Note that not all the subject metadata may be properly read in
+due to the fact that NK changes its specification
```
Comment (hoechenberger) [inline tutorials/io/plot_20_reading_eeg_data.py:204]:
```suggestion
due to the fact that NK changes the specification
```

**Review by hoechenberger — COMMENTED** [review]
Just some ideas on phrasing

**Review by hoechenberger — APPROVED** [review]

**Comment by larsoner** [comment]
Thanks @adam2392 !

**Pull request merged**